tabled_derive/src/lib.rs

```diff
@@ -477,6 +531,17 @@ fn use_function_no_args(function: &str) -> TokenStream {
     }
 }
 
+fn use_format(args: &TokenStream, custom_format: &str) -> TokenStream {
+    quote! { format!(#custom_format, #args) }
+}
+
+fn use_format_no_args(custom_format: &str, field: &TokenStream) -> TokenStream {
+    if custom_format.contains("{}") {
```

Comment:
just a strange if :sweat_smile:

Reply:
I guess it's error prone,
I'd guess we could just check arguments length and that's it?

Yes it would probably compile the code, I think.
But got `clippy` issue.

Reply:
I totally agree, thank you!